src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -95,6 +94,11 @@ trait Backend {
    */
   def stdoutStderr(descriptor: WorkflowDescriptor, callName: String, index: ExecutionIndex): CallLogs
 
+  /**
+   * Provides a function that given a WdlFile, returns its hash.
```

Comment:
Personally I think this is one of those spots where comments have a negative impact but YMMV and all that. I think it'd not have been confusing if I wasn't scanning it in a github diff, it's fine the way it is